src/utils/Chart.js

```diff
@@ -0,0 +1,35 @@
+const d3 = require('d3');
+
+const Chart = {
+  getDataMinMaxValues (data, axis) {
+    const max = d3.max(data, d => {
+      return Math.ceil(d[axis] / 1000) * 1000;
```

Comment:
Math.ceil will round up the number to the nearest decimal place so 569 becomes 569.  In this case I want it to round up to 1000 so I have to do the extra math.

![screen shot 2016-02-10 at 1 24 22 pm](https://cloud.githubusercontent.com/assets/6463914/12960672/b9c34c26-cff9-11e5-9ab4-b4b693f4c095.png)